libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts

```diff
@@ -59,13 +60,13 @@ describe('FormFieldComponent', () => {
     });
   });
 
-  fdescribe('When disabled', () => {
+  describe('When disabled', () => {
```

Comment:
Align casing with existing tests:
```suggestion
  describe('when disabled', () => {
```

Reply:
Done